By Daniel Widyanto 12/12/2008
   John Steele Scott 15/12/2006
Delete the location of the resource file. So people without c30_device.info can still
use the preprocessor

--- gcc-4.0.2/gcc-4.0.2/gcc/config/pic30/pic30.h.orig	2008-12-12 10:45:43.000000000 +0800
+++ gcc-4.0.2/gcc-4.0.2/gcc/config/pic30/pic30.h	2008-12-12 10:48:30.000000000 +0800
@@ -215,7 +215,7 @@
 */
 #define ASM_SPEC   "%{!.s:%{!.S:--relax}} %{mcpu=*:-p%*}"
 
-#define CC1_SPEC  "-mresource=%I-../../c30_device.info"
+#define CC1_SPEC  ""
 
 #define LINK_SPEC   "%{mcpu=*:-p%*}"
 
